lib/fathead/sass/parse_functions.py

```diff
@@ -171,6 +200,10 @@ def parse_for_data(self):
                 method_signature = self.parse_for_method_signature(function_section[0])
                 description = self.parse_for_description(function_section[1])
                 anchor = self.parse_for_anchor(function_section[0])
+                example = self.parse_for_example(function_section[0])
+
+                if example:
+                    description = description + '<br>'+ example
```

Comment:
@VeerpalBrar this is a good start, but I think it should really look more like what I described before: https://github.com/duckduckgo/zeroclickinfo-fathead/pull/253#discussion_r73779046

Do you have any thoughts on that?

I prefer having the text first, followed by the code snippet. We don't need to separate those with a `<br>` either as the `<p>` and `<pre>` have margins/padding